app/src/main/java/com/example/trainingintive/util/Constants.kt

```diff
@@ -0,0 +1,3 @@
+package com.example.trainingintive.util
+
+const val SIGN_IN_RESULT_CODE = 1001
```

Comment:
There is no need to create generic constant file. Keep constants in the file where they are used, in this case keep it in navigator.